allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureFixtureTest.java

```diff
@@ -21,11 +21,6 @@ public void allureFixtureTest() {
         step("Step inside allureFixtureTest");
     }
 
```

Comment:
What do we want to show in this test class?

I think it would be good to hint the main goal somewhere... like in string arguments of step invocations, or in some other way...

Here  goes at least one question, that I, as an allure user would ask if look at this example: 
- will the fixture (like before/after/...) be reported in allure if no step method is called inside?

Reply:
if yes... then this is an important info to reflect in this test class, i.e. define the fixture without inner steps, expecting them by default to appear in a report...

Reply:
UPDATE: my comments came from the some buggy behaviour in pytest in the past, that some steps was at wrong fixtures reported... that's why for me the behaviour was not obvious. Seems like for now we can ingore it...  Let's keep as it is...